### PR TITLE
Add /refugee/scores route, protected with basic auth

### PR DESCRIPTION
**Pull request by lucashu1**

### CODE REVIEW

**Comment by lucashu1** [comment]
For now, I'm using basic auth with an admin username and password that will be defined in the .env file. I know basic auth is kinda frowned upon, but this works for now, without the need to create a whole separate `users` table in SQL.

@Raghav-Maheshwari @QuinnNgo Lmk if you have any thoughts

File: src/server/index.js [hunk 1]
```diff
@@ -40,35 +45,15 @@ app.listen(PORT, () => {
...
-
-  // if there is no token
-  // return an error
-  return res.status(403).send({
-    success: false,
-    message: "No token provided."
-  });
-  
-}
+));
```
Comment (QuinnNgo) [inline src/server/index.js]: can we split this auth middleware into its own module?

File: src/server/index.js [hunk 2]
```diff
@@ -12,11 +12,12 @@ import donateRoutes from "./routes/donate";
 import itemsRoutes from "./routes/items";
 import storeRoutes from "./routes/stores";
 import currencyRoutes from "./routes/currency";
-import jwt from "jsonwebtoken";
 import cors from "cors";
 import bodyParser from "body-parser";
+const passport = require('passport');
+const BasicStrategy = require('passport-http').BasicStrategy;
```
Comment (QuinnNgo) [inline src/server/index.js]: please use `import` syntax (as opposed to `require`) for consistency

File: src/server/index.js (hunk 1 above)
Comment (lucashu1) [reply src/server/index.js]: Done

File: src/server/index.js (hunk 2 above)
Comment (lucashu1) [reply src/server/index.js]: Done

**Pull request merged**